java/mtsj/core/src/main/resources/db/specific/h2/V0001__R001_Create_schema.sql

```diff
@@ -199,4 +201,64 @@ CREATE TABLE RevInfo(
   id BIGINT NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1),
   "timestamp" BIGINT NOT NULL,
   userLogin VARCHAR(255)
+);
+
+-- *** Category_AUD ***
+CREATE CACHED TABLE PUBLIC.Category_AUD(
```

Comment:
For homogenity we should either qualify all tables with schema or none. Same applies for `CACHED` keyword that is omitted in all existing create table statements. We should do that consistently within the DDL.